src/core/simulation_results.jl

```diff
@@ -100,23 +100,21 @@ function SimulationResults(
                              )
 end
 
-function SimulationResults(sim::Simulation, stage_name::String; results_output_path = nothing)
-    simulation_store_path = joinpath(get_store_dir(sim), "data_store")
+function SimulationResults(sim::Simulation, stage_name::String)
```

Comment:
@claytonpbarrows and @daniel-thom under this scheme now we read the results directly from the simulation object